packages/transformers/image/src/ImageTransformer.js

```diff
@@ -21,12 +21,22 @@ export default (new Transformer({
   async transform({asset, options}) {
     asset.bundleBehavior = 'isolated';
 
-    let width = asset.query.width ? parseInt(asset.query.width, 10) : null;
-    let height = asset.query.height ? parseInt(asset.query.height, 10) : null;
-    let quality = asset.query.quality
-      ? parseInt(asset.query.quality, 10)
-      : undefined;
-    let format = asset.query.as ? asset.query.as.toLowerCase().trim() : null;
+    let width =
+      typeof asset.query.width === 'string'
+        ? parseInt(asset.query.width, 10)
+        : null;
```

Comment:
ugh that's annoying... should we even support duplicate query params?

Or maybe `asset.query` should be a `URLSearchParams` object?